firestore/main/package.json

```diff
@@ -12,12 +12,13 @@
   "license": "ISC",
   "dependencies": {
     "@google-cloud/firestore": "^4.4.0",
```

Comment:
OR queries where introduced in [v6.5.0](https://github.com/googleapis/nodejs-firestore/releases/tag/v6.5.0). We'll need to bump this :)

Reply:
Huh, I wonder why that's in here as a dependency. Seems like it should actually be removed - I tried updating it but it didn't change anything, and the Firebase admin sdk already [specifies `6.5.0` as a dep](https://github.com/firebase/firebase-admin-node/blob/de026a1a329c077ed22b2c1a3ee2fa324b4fb497/package.json#L209) (though it is optional??)

Reply:
I filed an issue for the missing re-export: https://github.com/firebase/firebase-admin-node/issues/2191

Reply:
A new version of the Admin SDK has been released that fixes this! https://github.com/firebase/firebase-admin-node/releases/tag/v11.9.0